### PR TITLE
Add OmniBTC

### PR DESCRIPTION
**Pull request by wbh1328551759**
Hi, we will participate the crowdloan auction .

### CODE REVIEW

**Review by jacogr — COMMENTED** [review]
I understand why the item is added to the Polkadot relay, unsure about the 2 to production and test - these are standalone without providers, so it adds no value.

**Comment by wbh1328551759** [comment]
I understand what you mean, but I'm sorry, I'm not sure what information I need to add is OK, so I'm following someone else's pr to submit the code

File: packages/apps-config/src/endpoints/productionRelayPolkadot.ts [hunk 1]
```diff
@@ -243,6 +243,16 @@ export const prodParasPolkadot: EndpointOption[] = [
...
     }
   },
+  {
+    info: 'omnibtc',
+    isUnreachable: true,
+    homepage: 'https://www.omnibtc.finance',
+    text: 'OmniBTC',
+    paraId: 2053,
+    providers: {
+      OmniBTC: 'wss://omnibtc.io/ws'
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/productionRelayPolkadot.ts:253]: We have 3 chains here with the exact same endpoint - this is not possible. It is either one of -

- a parachain
- a live network
- a test network

A single RPC cannot be linked to 3 different listings.

I would suggest just removing the entries in test networks and live networks and just keep it under Polkadot. (Which is where the chain lives)

Testing - these are solo test networks, running on their own
Production - these are solo production networks, running on their own

File: packages/apps-config/src/endpoints/production.ts [hunk 2]
```diff
@@ -207,6 +207,13 @@ export const prodChains: EndpointOption[] = [
       OnFinality: 'wss://nodle.api.onfinality.io/public-ws'
     }
   },
+  {
+    info: 'ominibtc',
+    text: 'OminiBTC',
+    providers: {
+      OmniBTC: 'wss://omnibtc.io/ws'
+    }
+  },
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/production.ts:216]:
```suggestion
```

Remove as per my comment below.

File: packages/apps-config/src/endpoints/testing.ts [hunk 3]
```diff
@@ -369,6 +369,14 @@ export const testChains: EndpointOption[] = [
...
     }
   },
+  {
+    info: 'omnibtc',
+    isDisabled: true, // https://github.com/polkadot-js/apps/issues/6635
+    text: 'OmniBTC',
+    providers: {
+      OmniBTC: 'wss://omnibtc.io/ws'
+    }
+  },
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testing.ts:379]:
```suggestion
```

Removed as per my comment above.

**Comment by wbh1328551759** [comment]
Okay, thanks for the guidance, I've removed useless content from testing and production

**Review by jacogr — APPROVED** [review]
Thank you!

**Pull request merged**